addons/account_peppol/models/account_move.py

```diff
@@ -63,3 +72,31 @@ def _compute_peppol_move_state(self):
                 move.peppol_move_state = 'ready'
             else:
                 move.peppol_move_state = move.peppol_move_state
+
+    @api.depends('invoice_line_ids.tax_ids', 'invoice_line_ids.product_id', 'partner_id.peppol_eas', 'partner_id.peppol_endpoint')
+    def _compute_peppol_warning(self):
+        for move in self:
+            if any([move.company_id.account_peppol_proxy_state != 'active',
+                    move.peppol_move_state in ('skipped', 'canceled', 'done'),
+                    move.partner_id.ubl_cii_format in (False, 'facturx'),
```

Comment:
To stay consistent with the send & print, you need to add oioubl here I think :thinking: